build-source-tarball.sh

```diff
@@ -173,6 +173,11 @@ find $TARBALL_ROOT/src \( -type f \( \
     -iname *.zip -o \
     -iname *.nupkg \) \) -exec rm {} \;
 
+#Remove binaries from tools-local as we don't want them to end up in tarball, the intent is to have them built within the context of tarball
```

Comment:
Maybe you want to remove the bin and obj directories from Tasks and Task.XPlat instead?  There are dlls and pdbs in the obj directory.

Reply:
Thanks for the catch.

Reply:
Could we remove these from the tarball copy instead like here? https://github.com/dotnet/source-build/blob/1e937de0b9e256bc9b4107b2e9ab3f3f9b067a69/build-source-tarball.sh#L206

Then we don't trash stuff from someone's working directory that we don't need to.